data/sql/updates/pending_db_world/rev_1591204377400264600.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1591204377400264600');
+
+-- Quest 3181 must not be taken for this gossip to take place
+UPDATE `creature_template` SET `gossip_menu_id`='50000' , `ScriptName`='' WHERE  `entry`=3836;
+
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (50000, 50001, 50002. 50003, 50004, 50005, 50006, 50007);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES 
+(50007, 1839),
+(50006, 1838),
+(50005, 1837),
+(50004, 1836),
+(50003, 1835),
+(50002, 1834),
+(50001, 1833),
+(50000, 1833);
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (50000, 50001, 50002. 50003, 50004, 50005, 50006, 50007);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES 
+(50003, 0, 0, 'Doesn\'t matter, i\'m invulnerable.', 0, 1, 1, 50004, 0, 0, 0, NULL, 0, 0),
+(50004, 0, 0, 'Yes...', 0, 1, 1, 50005, 0, 0, 0, NULL, 0, 0),
+(50005, 0, 0, 'Ok, i\'ll try to remember that.', 0, 1, 1, 50006, 0, 0, 0, NULL, 0, 0),
+(50006, 0, 0, 'A key? Ok!', 0, 1, 1, 0, 0, 0, 0, NULL, 0, 0),
+(50002, 0, 0, 'Ok, so what is this other way?', 0, 1, 1, 50003, 0, 0, 0, NULL, 0, 0),
+(50001, 0, 0, 'But i need to get there, now open the gate!', 0, 1, 1, 50002, 0, 0, 0, NULL, 0, 0),
+(50000, 0, 0, 'Open the gate please, i need to get to Searing Gorge', 0, 1, 1, 50001, 0, 0, 0, NULL, 0, 0),
+(50000, 1, 0, 'Umm... Pebblebitty... the gate is open.', 0, 1, 1, 50007, 0, 0, 0, NULL, 0, 0);
```

Comment:
Why are you deleting from `50000` to `50007` if you are using only from `50000` to `50007`?
Did you forget the text? because above in `gossip_menu` you are using `50007` too

Reply:
Somehow i forgot to add something to the SQL, I'll add in like an hour!